app/org/sagebionetworks/bridge/models/studies/Study.java

```diff
@@ -22,7 +22,7 @@
 
     public static final ObjectWriter STUDY_WRITER = new BridgeObjectMapper().writer(
         new SimpleFilterProvider().addFilter("filter", 
-        SimpleBeanPropertyFilter.serializeAllExcept("stormpathHref")));
+        SimpleBeanPropertyFilter.serializeAllExcept("stormpathHref", "active")));
```

Comment:
There's a test in DynamoStudyTest that tests this filter. Can we update this there?